addons/web/static/tests/helpers/test_utils_dom.js

```diff
@@ -12,6 +12,22 @@ var concurrency = require('web.concurrency');
  * testUtils file.
  */
 
+/**
+ * Returns a promise that will be resolved after the tick after the
+ * nextAnimationFrame
+ *
+ * This is usefull to guarantee that OWL has had the time to render
+ *
+ * @returns {Promise}
+ */
+function returnAfterNextAnimationFrame() {
```

Comment:
we need to discuss this:
- either we slow down the clicks and events so that owl has the time to do its job,
- or we accelerate owl (by patching requestAnimationFrame